myScript.js

```diff
@@ -0,0 +1,14 @@
+//edtior code
+//get textarea + apply codemirror library 
+var editor = CodeMirror.fromTextArea(document.getElementById('editor'), {
+	mode: "css",
+	theme: "xq-dark",
+	lineNumbers: true,
+	autoCloseBrackets: true
+});
+
+document.getElementById("editor").addEventListener("change", updateCss);
+
+function updateCss(){
+	document.getElementById("myCss").innerHTML = editor.value; 
```

Comment:
I can't seem to find any element tag with the id "myCss"... maybe if you tried adding a style tag with such id to your html page it could work

Reply:
I put a style tag in my html with that id but it does not seem to work :( thank you for the comment!